src/plots/cartesian/set_convert.js

```diff
@@ -600,16 +600,29 @@ module.exports = function setConvert(ax, fullLayout) {
         var rangebreaksIn = ax.rangebreaks || [];
         var bnds, b0, b1, vb, vDate;
 
+
+        if (!rangebreaksIn._cachedPatterns) {
+            rangebreaksIn._cachedPatterns = rangebreaksIn.map(brk => {
+                return brk.enabled && brk.bounds ? Lib.simpleMap(brk.bounds, brk.pattern ?
+                    cleanNumber :
+                    ax.d2c // case of pattern: ''
+                ) : null;
+            });
+        }
+        if(!rangebreaksIn._cachedValues) {
+            rangebreaksIn._cachedValues = rangebreaksIn.map(brk => {
```

Comment:
Thanks for the PR.
Please use ES5 syntax here:
```suggestion
            rangebreaksIn._cachedValues = rangebreaksIn.map(function(brk) {
```
so that the tests pass.